### PR TITLE
Get rid of early BigTest prototype

### PR DESCRIPTION
**Pull request by cherewaty**
## Purpose
We can now go all-in with BigTest `it()`s.

## Approach
Instead of the early BigTest prototype that was in this repo, we can import directly from the actual BigTest package. The former contents of `tests/it-will.js` are gone.

Two tests were falsely reporting before. It's pretty cool that those false positives were caught in this switch.

## Next Steps
- Refactor all page objects to use BigTest.
- Remove uses of `convergeOn()`, since we shouldn't have to use it directly.

## Screenshots
screen shot 2018-03-08 at 5 23 22 pm

### CODE REVIEW

File: tests/customer-resource-custom-coverage-test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-/* global describe, beforeEach */
+import { beforeEach, describe, it } from '@bigtest/mocha';
 import { expect } from 'chai';
-import it, { convergeOn } from './it-will';
+import { convergeOn } from '@bigtest/convergence';
```
Comment (wwilsman) [inline tests/customer-resource-custom-coverage-test.js]: One thing to note, `convergeOn` is [_technically_ internal](https://github.com/thefrontside/bigtest/tree/master/packages/convergence#convergeon) even though we export it.

If want to enforce this, we should probably convert these to `new Convergence().once()`. We frequently do `convergeOn(...).then()` which will need to be changed to `.do(...)`. If the `.then()` returns a promise, `.do()` supports that as well (though not officially).

Comment (wwilsman) [reply tests/customer-resource-custom-coverage-test.js]: Didn't quite read the whole PR before jumping the gun. 😆I was already reviewing before it got opened.

Comment (cherewaty) [reply tests/customer-resource-custom-coverage-test.js]: I'll make another PR to change over to that pattern.

Comment (cherewaty) [reply tests/customer-resource-custom-coverage-test.js]: https://github.com/folio-org/ui-eholdings/pull/268

**Pull request merged**